docs/rules/no-cb-test.md

```diff
@@ -0,0 +1,26 @@
+# Ensure no `test.cb()` is used
+
+Disallow the use of `test.cb()`.
```

Comment:
Sounds good! I disabled the rule by default though, as its not a rule that every codebase can afford to enforce.